assessments/guide/outline.md

```diff
@@ -52,35 +52,46 @@ for the remaider of this document). Describe the scenarios in which the project
 * Operation.  A description of the operational aspects of the system, such as
 how keys are likely to be managed and stored.
 
-## Project design
+## Project Design
 * Design.  A description of the system design that discusses how it works.
 This is especially critical for any mechanisms that relate to the security
-of the system.
+of the system.  Include architecture and network (if applicable) information such as encryption of traffic between services, access control types (RBAC, etc.) and enforcement, or security logging, etc.
 
-## Security analysis
-* Attacker motivations.  A discussion about the likely goals of an attacker.
+## Configuration and Set-Up
+* Default.  Documentation describing the default configuration of the project with initial set-up instructions (link to docs is acceptable). Documentation should identify potential security risks/trade-offs of the default config.
+* Secure. Documentation describing recommended secure configuration and set-up instructions, beyond defaults with justification for selection and trade-offs (link to docs is acceptable).
+* Advanced Secure. If applicable, documentation describing advanced settings for most hardened configuration of the project to include justification for selection and trade-offs (link to docs is acceptable).
+
+## Project Compliance
+* Compliance.  List any security standards or sub-sections the project is already documented as meeting (PCI-DSS, COBIT, ISO, GDPR, etc.).
+
```

Comment:
Really like the additions of config/setup and compliance.   I believe this will help projects think to write up relevant aspects of the project that they might otherwise forget to include.